amf-api-contract/shared/src/main/scala/amf/apicontract/internal/validation/shacl/graphql/GraphQLArgumentValidator.scala

```diff
@@ -34,12 +68,17 @@ object GraphQLArgumentValidator {
     val properties = node.properties.flatMap { prop =>
       validateDefaultValue(prop.datatype.getOrElse(""), prop.default, prop.property, prop.annotations)
     }
-    properties
+
+    val operations = node.operations.flatMap(_.parameters).flatMap { param =>
```

Comment:
Did I miss this?
I believe I removed this because I'm already validating these in AbstractParameters rather than starting from an Object. Left the above validation because in directives we are synthesizing an Object, but that Object will never have operations

Reply:
Suriprised this didn't alter any .report goldens